### ISSUE TITLE
nginx enchanced broken

### ISSUE THREAD

**Issue by todiadiyatmo** [szepeviktor/fix-w3tc#51]
Is anyone notice that the page enchanced is not working properly (or maybe just me)

```
set $w3tc_ext "";
if (-f "$document_root/wp-content/cache/page_enhanced/$http_host/$request_uri/_index$w3tc_ua$w3tc_rewrite.html") {
    set $w3tc_ext .html;
}
if (-f "$document_root/wp-content/cache/page_enhanced/$http_host/$request_uri/_index$w3tc_ua$w3tc_rewrite.xml") {
    set $w3tc_ext .xml;
}
if ($w3tc_ext = "") {
  set $w3tc_rewrite 0;
}
if ($w3tc_rewrite = 1) {
    rewrite .* "/wp-content/cache/page_enhanced/$http_host/$request_uri/_index$w3tc_ua$w3tc_rewrite$w3tc_ext" last;
}
````

the `$w3tc_rewrite` will have a value of `1` if rewrite should be performed (page exist in cache)
so by having `$w3tc_rewrite` in try files and rewrite section, the script will always fail.. because the file name don't have `1` on it .. .

for example for index page it is writen as `_index.html ` where the nginx script try to find `_index1.html `

**Comment by todiadiyatmo** [comment]
https://github.com/todiadiyatmo/fix-w3tc/commit/80b2f3d919412052055c1dbf6824d390448b5f2d

this patch seems fix the issue..

**Comment by todiadiyatmo** [comment]
i can also confirm (at least from my machine) the apache version is not working either.. i have mod_rewrite installed...

**Comment by nigrosimone** [comment]
@todiadiyatmo please, can you publish a clean PR here? this has so many modified line by white space. It's difficult understand what do you really change.

**Comment by amiga-500** [comment]
> i can also confirm (at least from my machine) the apache version is not working either.. i have mod_rewrite installed...

@todiadiyatmo is your apache version still not working?  Is your ngix side still working fine after your patch?

**Comment by todiadiyatmo** [comment]
@amiga-500 proper pull request created https://github.com/szepeviktor/fix-w3tc/pull/59

For apache version mine is not working ,but not really sure, i don't know much about apache rewrite :(
